component/tests/golden/minio/appcat/appcat/sli_exporter/90_slo_vshn_minio_ha.yaml

```diff
@@ -41,7 +41,7 @@ spec:
             sloth_window: 1h
           record: slo:sli_error:ratio_rate1h
         - expr: |
-            (sum(rate(appcat_probes_seconds_count{reason!="success", service="VSHNMinio", ha="true"}[2h]) or 0*rate(appcat_probes_seconds_count{service="VSHNMinio"}[2h])) by (service, namespace, name, organization, sla) or sum(0*rate(appcat_probes_seconds_count{service="VSHNMinio"}[2h])) by (service, namespace, name, organization, sla))
```

Comment:
This part of the query is redundant.

`0*rate(appcat_probes_seconds_count{service="VSHNMinio"}[2h])) by (service, namespace, name, organization, sla)` -> this part of the query will always produce a value of 0. This is necessary so it generates a value, otherwise it wouldn't even show up.

`sum(0*rate(appcat_probes_seconds_count{service="VSHNMinio"}[2h])) by (service, namespace, name, organization, sla)` -> does the exact same thing, except it sums it up. The sum of 0 is still 0, so it does not really do anything.

Reply:
of course, thanks for explanation